src/react-components/ui-root.js

```diff
@@ -700,7 +701,7 @@ class UIRoot extends Component {
       showFullScreenIfAvailable();
     }
 
-    this.props.enterScene(this.state.mediaStream, this.state.enterInVR, this.props.hubId);
+    this.props.enterScene(this.state.mediaStream, this.state.enterInVR, this.state.muteOnEntry);
```

Comment:
I have no idea what this last arg was doing here. AFAICT it was never used/referenced

Reply:
🤷‍♂ 🤷‍♀